web/src/resources/osk/kmwosk.css

```diff
@@ -295,19 +295,19 @@
 .tablet.android .kmw-key-deadkey{color:#0d0;background-color:#777;}
 
 .tablet.android .kmw-banner-bar {
-  background-color: #b4b4b8;
+  background-color: #222;
   position: relative;
   margin-top: 4px;
   height: 100%;
   width: 100%;
 }
```

Comment:
Making this change means we should be able to simplify kmwosk.css a bit via the following:

```css
.phone.android .kmw-banner-bar,
.tablet.android .kmw-banner-bar {
  background-color: #222;
  position: relative;
  margin-top: 4px;
  height: 100%;
  width: 100%;
}
```

There would be no need to have separate definitions for the two.

See line 185 for an existing CSS style definition using this pattern for iOS.